heartbeat/monitors/active/http/config.go

```diff
@@ -103,6 +114,20 @@ var defaultConfig = Config{
 	},
 }
 
+func (r *responseConfig) Validate() error {
+	switch strings.ToLower(r.IncludeBody) {
+	case "always", "on_error", "never":
+	default:
+		return fmt.Errorf("unknown option for `include_body`: '%s', please use one of 'always', 'on_error', 'never'", r.IncludeBody)
+	}
+
+	if r.IncludeBodyMaxBytes < 0 {
```

Comment:
```suggestion
	if r.IncludeBodyMaxBytes <= 0 {
```

0 bytes is also not useful I think?

Reply:
Yes, since you should just set `never` as the option. Good catch!